packages/utilities/package.json

```diff
@@ -35,6 +35,7 @@
   },
   "dependencies": {
     "axios": "^0.19.0",
+    "carbon-components": "^10.9.0",
     "js-cookie": "^2.2.1",
     "window-or-global": "^1.0.1"
   },
```

Comment:
Also for the prefix thing I'm using the `carbon-components` package as a dependency in the utilities package.json. Can anyone advise of this is the correct approach? This way it worked well in the test I made locally and automated tests passed fine too.

Reply:
Hmm, it makes sense to pull it in since it needs the settings for the prefix. It's just a little odd for a utility package to pull that in just for the prefix, lol! I don't think it is doing any harm though (at least I hope not)